lib/openhab/dsl/types/quantity.rb

```diff
@@ -171,6 +171,33 @@ def inspect
           end
         end
 
+        #
+        # Test if the value is positive
+        #
+        # @return [boolean] true if the quantity is > 0 in the same unit
+        #
+        def positive?
+          quantity > QuantityType.new(0, quantity.unit)
```

Comment:
Wouldn't it be more efficient to use `quantity.double_value > 0`? Or perhaps even `quantity.double_value.positive?` would work?

Reply:
Great idea! Thanks, I didn't think of that.